src/components/Layout/Layout.module.scss

```diff
@@ -0,0 +1,15 @@
+.Layout {
+  align-items: center;
+  background-image: url("../../../public/assets/images/bg-purple.jpeg");
```

Comment:
@ivanms1  
I noticed some compression artifacts in the gradient image so I thought we can consider replacing [/public/assets/images/bg-purple.jpeg](https://github.com/Free-Code-Camp-Seoul/free-code-camp-seoul.github.io/blob/e5f44d7caa558b7bbf5c39d297a6c1e500ea7eb1/public/assets/images/bg-purple.jpeg) with a CSS `linear-gradient` like this. I'm not sure about the gradient angle or exact hex color values, but this can be a starting point.

```suggestion
  background-image: linear-gradient(135deg, #211cab, #c74a95);
```

| `jpg` | `css` |
|--|--|
| <img width="869" alt="Screen Shot 2022-11-27 at 15 30 00" src="https://user-images.githubusercontent.com/24983797/204122468-a284e349-2e1c-4f4c-9942-969fd785399e.png"> | <img width="869" alt="Screen Shot 2022-11-27 at 15 29 40" src="https://user-images.githubusercontent.com/24983797/204122480-1b1f9da1-92e3-47cf-8a14-3e0f51871a8b.png"> |

Reply:
oh nice, yeah let's use that instead.